### PR TITLE
Remove remaining default select_relateds

### PR DESCRIPTION
**Pull request by phlax**
these should be added where required not as the default query

### CODE REVIEW

**Comment by unho** [comment]
The "add where required" part is meant to be part of a separate PR? Have you checked the impact on performance?

**Comment by phlax** [comment]
> Have you checked the impact on performance?

yep - massive improvement

File: pootle/apps/pootle_language/models.py [hunk 1]
```diff
@@ -108,7 +98,7 @@ class Language(models.Model, TreeItem):
     directory = models.OneToOneField('pootle_app.Directory', db_index=True,
                                      editable=False)
 
-    objects = LanguageManager()
+    objects = models.Manager()
```
Comment (unho) [inline pootle/apps/pootle_language/models.py]: Is it necessary to keep this line?

File: pootle/apps/pootle_language/models.py [hunk 2]
```diff
@@ -94,6 +83,7 @@ class Language(models.Model, TreeItem):
                           'localization-guide/en/latest/l10n/'
                           'pluralforms.html">'
                           'our page</a> on plural forms.')
+
```
Comment (unho) [inline pootle/apps/pootle_language/models.py]: Please remove this blank line.

File: pootle/apps/pootle_store/models.py [hunk 3]
```diff
@@ -1351,11 +1331,6 @@ def get_last_updated_info(self):
 class StoreManager(models.Manager):
```
Comment (unho) [inline pootle/apps/pootle_store/models.py]: It seems that now there is not a big difference between this and `simple_objects`, so perhaps `simple_objects` can be dropped as well.

Comment (phlax) [inline pootle/apps/pootle_store/models.py]: > It seems that now there is not a big difference between this and simple_objects, so perhaps simple_objects can be dropped as well.

probably - but that would require changing anywhere that currently uses - so i reckon that is a separate PR

File: pootle/apps/pootle_language/models.py (hunk 1 above)
Comment (phlax) [inline pootle/apps/pootle_language/models.py]: it seems that you have to if there is another manager

**Comment by unho** [comment]
lgtm

**Comment by julen** [comment]
As we [commented yesterday on IRC](https://botbot.me/freenode/pootle-dev/2015-12-16/?msg=56261218&page=1), in general this makes things faster except the cross-language resource pages (such as `/projects/<code>/resource/file.foo`). The different is not huge or noticeable for end-users, but it's worth mentioning it here for the record.

**Comment by phlax** [comment]
@julen thanks for this - i am following this up

**Pull request merged**